CHANGELOG

```diff
@@ -1,7 +1,5 @@
-[v#.#.#] ([MMM’YY])
-  * [feature #1]
-  * [feature #2]
-  * [feature ...]
+[v3.19.0] ([MMM’20])
+  * Improve accessibility
```

Comment:
I think for the changelog this might be worth expanding to show-off a bit of what was changed. Even if it's vague. Font contrast to meet x standards, label names, aria values etc etc. Let's everyone know we care.